modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -279,12 +280,23 @@ public function enqueue_block_editor_assets() {
 			true
 		);
 
+		// phpcs:ignore WordPress.Security.NonceVerification
+		$editor_deprecated               = isset( $_GET['editor/after-deprecation'] );
+		$classic_editor_plugin_available = is_plugin_inactive( 'classic-editor/classic-editor.php' );
+		$switch_visible                  = ( ! $editor_deprecated || jetpack_is_atomic_site() ) && $this->is_iframed_block_editor() && $classic_editor_plugin_available && current_user_can( 'activate_plugin' );
+
+		// The following is only to allow testing link without an atomic site.
+		// phpcs:ignore WordPress.Security.NonceVerification
+		if ( 'show' === $_GET['editor/after-deprecation'] ) {
```

Comment:
This may trigger PHP notices in some situations:

Undefined index: editor/after-deprecation
 Type: PHP Notice Line: 290
 File: wp-content/plugins/jetpack-dev/modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php